Source/HTTP/XMLHttp/xmlhttp_http_task.cpp

```diff
@@ -123,6 +122,7 @@ void xmlhttp_http_task::perform_async(
             return;
         }
 
+#if !HC_XDK_API
```

Comment:
why is this not needed on xdk?  looks like m_hRequestBodyStream is used below

Reply:
Yeah, working on fixing this next